src/NzbDrone.Core/Datastore/Migration/115_change_drone_factory_variable_name.cs

```diff
@@ -0,0 +1,20 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(115)]
+    public class change_drone_factory_variable_name : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Execute.Sql("DELETE FROM Config WHERE [Key] = 'downloadedepisodesfolder'");
+            Execute.Sql("DELETE FROM Config WHERE [Key] = 'downloadedepisodesscaninterval'");
```

Comment:
Do we need to add the album columns or are they already there?

Reply:
They get created automatically, config table is just a key/value table, so items don't get added until they are changed originally.